plutus-ledger-api/src/PlutusLedgerApi/V1.hs

```diff
@@ -105,52 +107,42 @@ module PlutusLedgerApi.V1 (
   DatumHash (..),
 
   -- * Data
-  PLC.Data (..),
-  BuiltinData (..),
-  ToData (..),
-  FromData (..),
-  UnsafeFromData (..),
-  toData,
-  fromData,
-  dataToBuiltinData,
-  builtinDataToData,
+  Common.Data (..),
+  Common.BuiltinData (..),
+  Common.ToData (..),
+  Common.FromData (..),
+  Common.UnsafeFromData (..),
+  Common.toData,
+  Common.fromData,
+  Common.unsafeFromData,
+  Common.dataToBuiltinData,
+  Common.builtinDataToData,
 
   -- * Errors
-  EvaluationError (..),
-  ScriptDecodeError (..),
+  Common.MonadError,
+  Common.EvaluationError (..),
+  Common.ScriptDecodeError (..),
 ) where
 
-import Data.SatInt
-import PlutusCore.Data qualified as PLC
-import PlutusCore.Evaluation.Machine.ExBudget as PLC
-import PlutusCore.Evaluation.Machine.ExMemory (ExCPU (..), ExMemory (..))
-import PlutusLedgerApi.Common as Common hiding (deserialiseScript, evaluateScriptCounting,
-                                         evaluateScriptRestricting)
-import PlutusLedgerApi.Common qualified as Common (deserialiseScript, evaluateScriptCounting,
-                                                   evaluateScriptRestricting)
-import PlutusLedgerApi.V1.Address
-import PlutusLedgerApi.V1.Bytes
-import PlutusLedgerApi.V1.Contexts
-import PlutusLedgerApi.V1.Credential
-import PlutusLedgerApi.V1.Crypto
-import PlutusLedgerApi.V1.DCert
-import PlutusLedgerApi.V1.EvaluationContext
-import PlutusLedgerApi.V1.Interval hiding (singleton)
-import PlutusLedgerApi.V1.ParamName
+import PlutusLedgerApi.Common qualified as Common
+import PlutusLedgerApi.V1.Address qualified as Address
+import PlutusLedgerApi.V1.Bytes qualified as Bytes
+import PlutusLedgerApi.V1.Contexts qualified as Contexts
+import PlutusLedgerApi.V1.Credential qualified as Credential
+import PlutusLedgerApi.V1.Crypto qualified as Crypto
+import PlutusLedgerApi.V1.DCert qualified as DCert
+import PlutusLedgerApi.V1.EvaluationContext qualified as EvaluationContext
+import PlutusLedgerApi.V1.Interval qualified as Interval
+import PlutusLedgerApi.V1.ParamName qualified as ParamName
 import PlutusLedgerApi.V1.Scripts as Scripts
-import PlutusLedgerApi.V1.Time
-import PlutusLedgerApi.V1.Value
-import PlutusTx (FromData (..), ToData (..), UnsafeFromData (..), fromData, toData)
-import PlutusTx.Builtins.Internal (BuiltinData (..), builtinDataToData, dataToBuiltinData)
-import PlutusTx.Prelude (BuiltinByteString, fromBuiltin, toBuiltin)
-
-import Control.Monad.Except (MonadError)
```

Comment:
This module now contains only `PlutusLedgerApi` imports, no Plutus Tx or `base`.